packages/vkui/src/components/CustomSelect/CustomSelect.tsx

```diff
@@ -148,6 +163,15 @@ export interface SelectProps extends NativeSelectProps, FormFieldProps, TrackerO
   onClose?: VoidFunction;
   onOpen?: VoidFunction;
   icon?: React.ReactNode;
+  /**
+   * Кастомная кнопка для очистки значения.
+   * Должна принимать обязательное свойство `onClick`
+   */
+  clearButton?: React.ComponentType<CustomSelectClearButtonProps>;
```

Comment:
Только
```suggestion
  ClearButton?: React.ComponentType<CustomSelectClearButtonProps>;
```
т.к. это конструктор